joplin/base/templatetags/joplin_tags.py

```diff
@@ -32,7 +34,11 @@ def get_revision_preview_url(*args, **kwargs):
 @register.simple_tag
 def get_style_guide_url(*args, **kwargs):
     content_type = kwargs['content_type'].name
-    return os.environ['STYLEGUIDE_URL'] + STYLEGUIDE_PAGES[content_type]
+    try:
+        get_style_guide_url = os.environ['STYLEGUIDE_URL'] + STYLEGUIDE_PAGES[content_type]
+    except Exception as e:
```

Comment:
gives a default fallback for a page if its missing a styleguide url

Reply:
You mean if its missing a STYLEGUIDE_PAGES[content_type] ?

Reply:
Yes